console/console-init/ui/src/containers/AdapterListContainer/AdapterListContainer.tsx

```diff
@@ -14,24 +14,33 @@ export const AdapterListContainer: React.FC<{ id: string }> = ({ id }) => {
   const { projectname } = useParams();
 
   const queryResolver = `
-  iotProjects {
+  objects {
+    ... on IoTProject_iot_enmasse_io_v1alpha1 {
     endpoints{
       name
       url
       host
       port
       tls
     }
+  }
   }`;
 
   const { data } = useQuery<IIoTProjectsResponse>(
     RETURN_IOT_PROJECTS({ projectName: projectname }, queryResolver)
   );
 
   const { allProjects } = data || {
-    allProjects: { iotProjects: [] }
+    allProjects: { objects: [] }
   };
-  const adapters: IAdapter[] = [];
+  let adapters: IAdapter[] = [];
+  if (
```

Comment:
can we avoid if consition if not required.
adapters = allProjects?.objects?[0]?.endpoints;